tests/unit/util/test_configuration.py

```diff
@@ -10,6 +10,7 @@
 import pytest
 import responses
 from attrs import asdict
+from attrs.exceptions import FrozenInstanceError
```

Comment:
this is an unused import. Do you want to add more tests here to test for this error?